index.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+//1. Задача на розворот числа:
+
+const currentMaxValue = 4589;
+let reverseMaxValue;
+
+const arrayOfValue = Array.from(String(currentMaxValue));
+const back = arrayOfValue.reverse();
+reverseMaxValue = back.join('');
+reverseMaxValue = Number(reverseMaxValue);
```

Comment:
```suggestion
reverseMaxValue = Number(Array.from(String(currentMaxValue)).reverse().join(''));
```

Там де змінна використовується один раз, можливо краще її взагалі не створювати